### ISSUE TITLE
Missing dependencies for Python 2.7

### ISSUE THREAD

**Issue by mnonnenmacher** [nexB/python-inspector#62]
python-inspector is missing some dependencies when used with `--python-version 27`.

Test project used: https://github.com/spdx/tools-python/tree/a48022e65a8897d0e4f2e93d8e53695d2c13ea23

When running `pip install` with Python 2.7 on this project the following packages are installed:

```shell
Successfully installed isodate-0.6.1 ply-3.11 pyparsing-2.4.7 rdflib-5.0.0 six-1.16.0 spdx-tools-0.5.4
```

But running `python-inspector --python-version 27 setup.py` only returns:

```json
[
  {
    "key": "ply",
    "package_name": "ply",
    "installed_version": "3.11",
    "dependencies": []
  },
  {
    "key": "rdflib",
    "package_name": "rdflib",
    "installed_version": "5.0.0",
    "dependencies": []
  },
  {
    "key": "six",
    "package_name": "six",
    "installed_version": "1.16.0",
    "dependencies": []
  }
]
```

So isodate-0.6.1 and pyparsing-2.4.7 are missing from the output.

**Comment by pombredanne** [comment]
Thanks for the report. Are you really still using this SPDX library with Python 2.7 in particular?

**Comment by sschuberth** [comment]
> Are you really still using this SPDX library with Python 2.7 in particular?

ORT is not really making use of that library. We just use that library as a test project for ORT's functional analyzer tests.

**Comment by pombredanne** [comment]
@sschuberth ack. Fair enough. Python 2 is not the key issue, but rather that they do compute various arguments at install time
The way to get these (without installing) will be a Python evaluation the provided values as they cannot be reliably parsed in this case. IMHO the best would be guard this with an `--insecure` CLI option. (NB: ORT behaviour prior to using python-inspector is what I would call "insecure" too, FWIW)

**Comment by sschuberth** [comment]
> NB: ORT behaviour prior to using python-inspector is what I would call "insecure" too, FWIW

Yes, I guess we aim for correctness over security 😉

**Comment by mnonnenmacher** [comment]
I agree that having this as a command line option would be good, then the decision would be up to the user.
